railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -37,3 +37,11 @@
 
 # Set both the `:open_timeout` and `:read_timeout` values for `:smtp` delivery method.
 # Rails.application.config.action_mailer.smtp_timeout = 5
+
+# Sign direct uploads with a specific purpose.
+#
+# Direct uploads should now be signed with a `:direct_upload` purpose. Previously, they were signed with `:blob_id`.
+# If you use direct uploads, your app will need to temporarily be able to find both signed types.
+# Make sure your entire app is migrated and stable on 7.0 before using this setting.
```

Comment:
Should there be a changelog for this?

I'm also curious about:

> If you use direct uploads, your app will need to temporarily be able to find both signed types.
> Make sure your entire app is migrated and stable on 7.0 before using this setting.

If we're going to go down this route of breaking uploads, we should have guidance on how to migrate. IMO